pkg/util/test/util.go

```diff
@@ -1159,6 +1219,76 @@ func validateCsiContainerInPxPods(namespace string, csi bool, timeout, interval
 	return nil
 }
 
+func validatePvcControllerPorts(annotations map[string]string, pvcControllerDeployment *appsv1.Deployment, timeout, interval time.Duration) error {
+	logrus.Debug("Validate PVC Controller custom ports")
+
+	if annotations == nil {
+		return nil
+	}
+
+	pvcPort := annotations["portworx.io/pvc-controller-port"]
```

Comment:
Starting k8s 1.22 we don't honor the `pvc-controller-port` annotation. Only `secure-port` annotation is honored. That is because the new pvc controller doesn't take the port parameter now.
Let's remove this port validation and keep just the secure-port, else this will start failing soon on k8s 1.22 and above.

Reply:
Removed `pvc-controller-port` from test and validation